frontend/src/queries/nodes/InsightViz/utils.ts

```diff
@@ -106,6 +106,22 @@ export const getShowValueOnSeries = (query: InsightQueryNode): boolean | undefin
     }
 }
 
+export const getShowPercentStackView = (query: InsightQueryNode): boolean | undefined => {
+    if (PERCENT_STACK_VIEW_DISPLAY_TYPE.includes(getDisplay(query) as ChartDisplayType)) {
```

Comment:
> Resolved: Also only trends insights support the percent stack view (see below how you implemented formatYAxisLabel)
Resolved: I think there is also a bug since this only considers the case where the query has a display set, but not the default display.

About the above two points, I understood the first point, the second point was not that clear to me.